CHANGELOG.md

```diff
@@ -893,7 +893,7 @@ fix rather than a breaking change in semantic versioning.
 **Internal changes**:
 
 - Specify 1.60.0 as minimum Rust version (@max-sixty, #1011)
-- Remove old `wee-alloc` code (@max-sixty, #1013)
+- Remove old `we-alloc` code (@max-sixty, #1013)
```

Comment:
This incorrect change must be stopped.

Reply:
Yeah, we should not auto correct typos.

Reply:
It's solved so many typos though! It very rarely makes mistakes... 

I'll exclude this one...